src/components/input-field/examples/input-field-text-multiple.tsx

```diff
@@ -0,0 +1,65 @@
+import { Component, h, State } from '@stencil/core';
+
+/**
+ * Multiple Fields
+ */
+@Component({
+    tag: 'limel-example-input-field-text-multiple',
+    shadow: true,
+    styleUrl: 'input-field-text-multiple.scss',
+})
+export class InputFieldTextExample {
+    @State()
+    private firstValue;
+
+    @State()
+    private secondValue;
+
+    @State()
+    private addDistance: boolean = false;
+
+    constructor() {
+        this.firstOnChange = this.firstOnChange.bind(this);
+        this.secondOnChange = this.secondOnChange.bind(this);
+        this.toggleMode = this.toggleMode.bind(this);
+    }
+
+    public render() {
+        return (
+            <div class={{ 'add-distance': this.addDistance }}>
+                <section>
+                    <limel-input-field
+                        label="Fields shouldn't be too close!"
+                        value={this.firstValue}
+                        onChange={this.firstOnChange}
+                    />
+                    <limel-input-field
+                        label="Type something here now to see why…"
+                        helperText="See how label covers the previous field? Now add some distance 👇"
+                        value={this.secondValue}
+                        onChange={this.secondOnChange}
+                    />
+                </section>
+                <limel-flex-container justify="start">
+                    <limel-checkbox
+                        label="Then click this to add distance between fields"
```

Comment:
Ha! Nice example! 👏

When I first saw it, I was just scrolling past to check something else, and assumed the old example had broken. Then I saw you commit message and thought "that doesn't seem very useful at all", thinking that you had just stacked two inputs on top of each other.

Then I actually looked closer at the example 😄 Very nice! 👌

Reply:
😊 Thank you Dr. Shchmitz! 😎